civiform/civiform
# Move eligibility gating toggle to be with the rest of the program settings

### Description

This moves the eligibility gating toggle to be with the rest of the program settings.

Previously, it was on a separate settings menu that was only accessible from these places:

**Program dashboard -> overflow menu on draft program -> "Settings":**
  
![image](https://github.com/civiform/civiform/assets/147177873/fdc8f8cb-0f72-46d2-be35-7f0bfb5f89ef)

**Program edit blocks page -> "program settings" link inside the helper text for eligibility conditions:**

![image](https://github.com/civiform/civiform/assets/147177873/50730cf0-ddca-4168-aaf6-0386137adcd8)
  

Neither of those locations are obvious to admins. This PR moves it to be with the rest of the program-level settings so it's easier to access.

## Release notes

### Checklist

#### General

Read the full guidelines for PRs [here](https://github.com/civiform/civiform/wiki/Technical-contribution-guide#creating-a-pull-request)



#### User visible changes


### Instructions for manual testing

1. Create a new program, set its eligibility to **non-gating**. Create a question with an eligibility condition. As an applicant, fill out the program with an eligible answer and verify you can submit the form. Fill out the form with a *not* eligible answer and verify you can still submit the form.
2. Create a new program, set its eligibility to **gating**. Create a question with an eligibility condition. As an applicant, fill out the program with an eligible answer and verify you can submit the form. Fill out the form with a *not* eligible answer and verify you get the "your answers aren't eligible" page and you cannot submit the form.
3. Change an existing program from gating to non-gating or vice versa. Verify the form behavior changes appropriately.

### Issue(s) this completes

Fixes #5852
